components/Result.tsx

```diff
@@ -1,9 +1,19 @@
 import { Album } from "@/app/utils/discogs.schemas";
+import Image from "next/image";
 
 export default function Result({ album }: { album: Album }) {
   return (
-    <>
-      <h3>Year: {album.year}</h3>
-    </>
+    <div className="flex mt-16">
```

Comment:
you can definitely go a little harder on the styles here if you want. Since the PR is for styling the album results, I'd go ahead and get these styles pretty close to how you want them to look in the end (not including the buttons of course).

Reply:
I have updated some styles and will continue to do so throughout this PR.